SonarRunner.Shim/PropertiesFileGenerator.cs

```diff
@@ -204,6 +207,33 @@ private static string TryGetCodeCoverageReport(ProjectInfo project, ILogger logg
             return vsCoverageReport;
         }
 
+        private static AnalysisProperties GetAnalysisProperties(AnalysisConfig config, ILogger logger)
+        {
+            AnalysisProperties properties = config.LocalSettings ?? new AnalysisProperties();
+
+            // There are some properties we want to override regardless of what the user sets
+            AddOrSetProperty(VSBootstrapperPropertyKey, "false", properties, logger);
+
+            return properties;
+        }
+
+        private static void AddOrSetProperty(string key, string value, AnalysisProperties properties, ILogger logger)
+        {
+            Property property;
+            Property.TryGetProperty(key, properties, out property);
+            if (property == null)
+            {
+                logger.LogDebug(Resources.MSG_SettingAnalysisProperty, key, value);
+                property = new Property() { Id = key, Value = value };
+                properties.Add(property);
+            }
+            else
+            {
+                logger.LogWarning(Resources.WARN_OverridingAnalysisProperty, key, value);
```

Comment:
This will give a warning even if the property has the same value, in which case we aren't really overriding.